src/xversionkeys.h

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2020 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+#ifndef BITCOIN_XVERSIONKEYS_H
+#define BITCOIN_XVERSIONKEYS_H
+
+#include <unordered_map>
+
+#define EXP_VER_PREFIX      0x0000
+#define BCHN_PREFIX         0x0001
+#define BU_PREFIX           0x0002
+
+
+/// old header
+
+
+namespace XVer
+{
+
+enum {
+                   BU_ELECTRUM_SERVER_PORT_TCP = 0x000000000002f00dUL,
+           BU_ELECTRUM_SERVER_PROTOCOL_VERSION = 0x000000000002f00eUL,
+                  BU_GRAPHENE_FAST_FILTER_PREF = 0x0000000000020004UL,
+             BU_GRAPHENE_MAX_VERSION_SUPPORTED = 0x0000000000020001UL,
+             BU_GRAPHENE_MIN_VERSION_SUPPORTED = 0x0000000000020005UL,
+                                BU_LISTEN_PORT = 0x0000000000020000UL,
+               BU_MEMPOOL_ANCESTOR_COUNT_LIMIT = 0x0000000000020009UL,
+                BU_MEMPOOL_ANCESTOR_SIZE_LIMIT = 0x000000000002000aUL,
+             BU_MEMPOOL_DESCENDANT_COUNT_LIMIT = 0x000000000002000bUL,
+              BU_MEMPOOL_DESCENDANT_SIZE_LIMIT = 0x000000000002000cUL,
+                               BU_MEMPOOL_SYNC = 0x0000000000020006UL,
+         BU_MEMPOOL_SYNC_MAX_VERSION_SUPPORTED = 0x0000000000020008UL,
+         BU_MEMPOOL_SYNC_MIN_VERSION_SUPPORTED = 0x0000000000020007UL,
+                        BU_MSG_IGNORE_CHECKSUM = 0x0000000000020002UL,
+                          BU_TXN_CONCATENATION = 0x000000000002000dUL,
+                              BU_XTHIN_VERSION = 0x0000000000020003UL,
+}; // enum keys
```

Comment:
rather than ordering by alphabetical i think it would be good to keep the ordering by hex value...IMO i think it makes it much easier for someone to add another value at then end when the time comes.

Reply:
yeah thats fine. i got this file by having the python script generate it before i removed the script. I can change that ordering.

Reply:
fixed